### ISSUE TITLE
kubeadm reset no longer prints iptables commands

### ISSUE THREAD

**Issue by krisdock** [kubernetes/kubeadm#2013]
## Is this a BUG REPORT or FEATURE REQUEST?

Choose one: FEATURE REQUEST

## Versions

**kubeadm version** (use `kubeadm version`): v1.17.2
**Environment**:
- **Kubernetes version** (use `kubectl version`): v1.17.2
- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release): Ubuntu 18.04
- **Kernel** (e.g. `uname -a`):
- **Others**:

## What happened?
`kubeadm reset` used to print out the strings of iptables commands for flushing all rules, but no longer does in v1.17.2. It was useful having them printed out following a reset for easy copy/paste.

**Comment by boluisa** [comment]
this is very useful command.  not cleaning up iptables has been  a common cause of cluster issues for a lot of users.

**Comment by neolit123** [comment]
hi, we found that users can end up playing with the command without much experience with iptables rules and causing deletion of unwanted items!

therefore we removed the command from the output, see:
https://github.com/kubernetes/kubernetes/commit/0d732009cc75f8333ad131f58dbcd6535c11ca3a#diff-681222fdc03e967cc65bfb7ac01b30c9

i'm going to have to close https://github.com/kubernetes/kubernetes/pull/87780 too.
thanks for understanding.
